src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -361,7 +361,7 @@ class DocBaseViewer extends BaseViewer {
         const previousPageButtonEl = this.containerEl.querySelector('.bp-previous-page');
         const nextPageButtonEl = this.containerEl.querySelector('.bp-next-page');
 
-        // Safari disables keyboard input in fullscreen
+        // Safari disables keyboard input in fullscreen before Safari 10.1
```

Comment:
I'll do that in a separate PR since it's not trivial to select 10.1 on iOS and Safari at the same time.